<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Mining the genome</title>
    <meta charset="utf-8" />
    <meta name="author" content="Ivan Voronin, Université Laval" />
    <script src="libs/kePrint/kePrint.js"></script>
    <link href="libs/lightable/lightable.css" rel="stylesheet" />
    <script src="libs/jquery-3.6.0.min.js"></script>
    <script src="libs/bootstrap.bundle.min.js"></script>
    <script src="https://kit.fontawesome.com/774d3477f3.js" crossorigin="anonymous"></script>
    
    <link rel="stylesheet" href="libs/bootstrap.min.css">
    <link rel="stylesheet" href="libs/animate.css">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Sans">
    
    <script src="libs/carousel.js"></script>
    <script src="libs/animate-slides.js"></script>
    
    <link rel="stylesheet" href="my-slides.css">
    
    <link rel="icon" type="image/x-icon" href="svg2/favicon.svg">
    <link rel="stylesheet" href="my-slides.css" type="text/css" />
    <link rel="stylesheet" href="libs/animate.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">

class: title-slide

&lt;script&gt;
$(document).ready(function() {
    $('#svg-lib').prependTo('body');
    $('#navig-links').appendTo('.remark-slide-content');
});
&lt;/script&gt;

&lt;div id='svg-lib' style='width:0;height:0;'&gt;
<!-- 
Author: Ivan Voronin
https://github.com/IvanVoronin

Feb 24, 2022
License: CC BY-NC-SA 4.0 https://creativecommons.org/licenses/by-nc-sa/4.0/ 
-->
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1"
    viewBox="0 0 4000 3000" stroke='white' font-family='sans'>
    <!-- 300px 300px -->
    <defs>
        <filter id="turned-off-glow" filterUnits="userSpaceOnUse" x="-50%" y="-50%" width="200%" height="200%">
            <feColorMatrix in="SourceGraphic" type="matrix" values="1 0 0 0 0
                        0 1 0 0 0
                        0 0 1 0 0
                        0 0 0 0.3 0" />
        </filter>
        <filter id="neon-glow" filterUnits="userSpaceOnUse" x="-50%" y="-50%" width="200%" height="200%">
            <!-- https://stackoverflow.com/questions/54112231/is-it-possible-to-create-a-glow-effect-in-svg -->
            <!-- blur the text at different levels-->
            <feGaussianBlur in="SourceGraphic" stdDeviation="5" result="blur5" />
            <feGaussianBlur in="SourceGraphic" stdDeviation="10" result="blur10" />
            <feGaussianBlur in="SourceGraphic" stdDeviation="20" result="blur20" />
            <feGaussianBlur in="SourceGraphic" stdDeviation="30" result="blur30" />
            <feGaussianBlur in="SourceGraphic" stdDeviation="50" result="blur50" />
            <!-- merge all the blurs except for the first one -->
            <feMerge result="blur-merged">
                <feMergeNode in="blur5" />
                <feMergeNode in="blur10" />
                <feMergeNode in="blur20" />
                <feMergeNode in="blur30" />
                <feMergeNode in="blur50" />
            </feMerge>
            <feMerge result="source-merged">
                <feMergeNode in="SourceGraphic" />
            </feMerge>

           <feColorMatrix result='blur-merged-light' in="blur-merged" type="matrix" values="
            1 0 0 0 0
            0 1 0 0 0
            0 0 1 0 0
            0 0 0 0.5 0" />

            <feColorMatrix in='SourceGraphic' result='source-dodged' type='matrix' values='
            1 1 1 0 0
            1 1 1 0 0
            1 1 1 0 0
            0 0 0 1 0' />

            <feMerge>
                <feMergeNode in="blur-merged-light" />
                <feMergeNode in="source-dodged" />
            </feMerge>
 
        </filter>

        <filter id="soft-glow" filterUnits="userSpaceOnUse" x="-50%" y="-50%" width="200%" height="200%">
            <!-- https://stackoverflow.com/questions/54112231/is-it-possible-to-create-a-glow-effect-in-svg -->
            <!-- blur the text at different levels-->
            <feGaussianBlur in="SourceGraphic" stdDeviation="5" result="blur5" />
            <feGaussianBlur in="SourceGraphic" stdDeviation="10" result="blur10" />
            <feGaussianBlur in="SourceGraphic" stdDeviation="20" result="blur20" />
            <!-- merge all the blurs except for the first one -->
            <feMerge result="blur-merged">
                <feMergeNode in="blur5" />
                <feMergeNode in="blur10" />
                <feMergeNode in="blur20" />
            </feMerge>
            <feMerge result="source-merged">
                <feMergeNode in="SourceGraphic" />
            </feMerge>

           <feColorMatrix result='blur-merged-light' in="blur-merged" type="matrix" values="
            1 0 0 0 0
            0 1 0 0 0
            0 0 1 0 0
            0 0 0 0.5 0" />

            <feColorMatrix in='SourceGraphic' result='source-dodged' type='matrix' values='
            1 1 1 0 0
            1 1 1 0 0
            1 1 1 0 0
            0 0 0 1 0' />

            <feMerge>
                <feMergeNode in="blur-merged-light" />
                <feMergeNode in="source-dodged" />
            </feMerge>
 
        </filter>

        <g id='frame'>
            <circle cx="150" cy="150" r="150" stroke-width="5" />
            <circle id='inner-frame' cx="150" cy="150" r="135" stroke-width="2.5" />
        </g>

        <path id='double-helix-shape'
            d="M 150 0 q -75 30 0 60 t 0 60 t 0 60 t 0 60 t 0 60 t 0 60 m 0 -50 q 75 -30 0 -60 t 0 -60 t 0 -60 t 0 -60 t 0 -60"
            stroke-width="3px" />

        <clipPath id='frame-clip'>
            <use href='#inner-frame' />
        </clipPath>

        <clipPath id='double-helix-clip'>
            <use href='#double-helix-shape' />
        </clipPath>

        <pattern id='helix-stripes' patternUnits='userSpaceOnUse' x='0' y='0' width='100' height='11.5'
            viewBox='0 0 100 10'>
            <path d="M 0 0 l 100 0" stroke-width='1.5px' />
        </pattern>

        <g id='double-helix' transform='translate(-50)'>
            <use href='#double-helix-shape' fill='url(#helix-stripes)' />
        </g>

        <g id='chip' transform='translate(70 30) scale(0.9)'>
            <path d="M 105 195 l 90 0 l 0 -120 l -25 -15 l -40 0 l -25 15 l 0 120
            M 105 100 l 90 0" stroke-width='2px' />
            <rect x='120' y='120' width='60' height='60' stroke-width='2' />
            <rect x='130' y='130' width='40' height='40' stroke-width='1.5px' fill='white' fill-opacity='0.2' />
        </g>

        <g id='lollipops'>
            <circle cx='180' cy='60' r='15' stroke-width='2.5' />
            <path d="M 160 60 l -30 0" stroke-width='2' />
            <circle cx='225' cy='93' r='15' stroke-width='2.5' />
            <path d="M 205 93 l -60 0" stroke-width='2' />
            <circle cx='165' cy='138' r='15' stroke-width='2.5' />
            <path d="M 145 138 l -19 0" stroke-width='2' />
            <circle cx='240' cy='196' r='15' stroke-width='2.5' />
            <path d="M 220 196 l -80 0" stroke-width='2' />
            <circle cx='180' cy='231' r='15' stroke-width='2.5' />
            <path d="M 160 231 l -30 0" stroke-width='2' />
        </g>

        <g id='ruler'>
            <rect x='200' y='75' width='30' height='150' stroke-width='2.5px' rx='5' />
            <path
                d="M 200 80 l 5 0 m -5 10 l 5 0 m -5 10 l 5 0 m -5 10 l 5 0 m -5 10 l 5 0 m -5 10 l 5 0 m -5 10 l 5 0 m -5 10 l 5 0 m -5 10 l 5 0 m -5 10 l 5 0 m -5 10 l 5 0 m -5 10 l 5 0 m -5 10 l 5 0 m -5 10 l 5 0 m -5 10 l 5 0"
                stroke-width='1.5px' />
        </g>

        <g id='pheno' transform='translate(-150 -150)'>

            <g clip-path='url(#frame-clip)' stroke-linecap='round' stroke-linejoin='round'>
                <g transform='translate(10)'>
                    <rect x='65' y='75' width='120' height='170' rx='5' stroke-width='2.5' />
                    <g id='item'>
                        <circle cx='87' cy='100' r='7' stroke-width='1.5px' />
                        <path d="M 105 100 l 65 0" stroke-width='1.5px' />
                    </g>
                    <use href='#item' transform='translate(0 30)' />
                    <use href='#item' transform='translate(0 60)' />
                    <use href='#item' transform='translate(0 90)' />
                    <use href='#item' transform='translate(0 120)' />
                    <path id='tick' d="M 86 103 l -7 -10 M 86 103 l 10 -16" stroke-width='1.5px' />
                    <use href='#tick' transform='translate(0 60)' />
                    <use href='#tick' transform='translate(0 90)' />
                </g>
                <path d="M 235 120 l 0 100 l -6 0 l 0 -100 l 6 0 l -3 -8 l -3 8" stroke-width='1.5'
                    stroke-linecap='round' stroke-linejoin='round' />
            </g>
            <use href='#frame' />
        </g>

        <g id='geno' stroke-linecap='round' stroke-linejoin='round' transform='translate(-150 -150)'>
            <g clip-path='url(#frame-clip)'>
                <use href='#double-helix' />
                <use href='#chip' />
            </g>
            <use href='#frame' />
        </g>

        <g id='sumstats' stroke-linecap='round' stroke-linejoin='round' transform='translate(-150 -150)'>
            <g clip-path='url(#frame-clip)'>
                <use href='#double-helix' />
                <use href='#lollipops' />
            </g>
            <use href='#frame' />
        </g>

        <g id='prs' stroke-linecap='round' stroke-linejoin='round' transform='translate(-150 -150)'>
            <g clip-path='url(#frame-clip)'>
                <use href='#double-helix' />
                <use href='#ruler' />
            </g>
            <use href='#frame' />
        </g>

        <!--
        <g id='classic-gwas-old' transform='translate(-300 -150)'>
            <mask id='classic-gwas-lab'>
                <rect x='0' y='0' width='600' height='300' fill='white' />
                <text x='300' y='150' fill='black' text-anchor='middle' alignment-baseline='middle' font-size='56px'
                    font-weight='bold'>
                    Classic GWAS
                </text>
            </mask>
            <rect x='0' y='0' width='600' height='300' mask='url(#classic-gwas-lab)' stroke-width='0' fill='white'
                opacity='0.3' rx='20' />
        </g>
        -->

        <g id='classic-gwas' transform='translate(-300 -150)'>
            <rect x='0' y='0' width='600' height='300' stroke-width='5px' rx='20px' />
            <rect x='15' y='15' width='570' height='270' stroke-width='2.5px' rx='5px' />

            <text x='300' y='150' fill='white' text-anchor='middle' dominant-baseline='middle' font-size='56px'
                font-weight='bold'>
                Classic GWAS
            </text>
        </g>

        <g id='meta-gwas' transform='translate(-500 -200)'>
            <rect x='0' y='0' width='1000' height='400' stroke-width='5px' rx='20px' />
            <rect x='15' y='15' width='970' height='370' stroke-width='2.5px' rx='5px' />
            <text x='500' y='200' fill='white' text-anchor='middle' dominant-baseline='middle' font-size='84px'
                font-weight='bold'>
                Meta-GWAS
            </text>
        </g>

        <g id='genomic-sem' transform='translate(-500 -200)'>
            <rect x='0' y='0' width='1000' height='400' stroke-width='5px' rx='20px' />
            <rect x='15' y='15' width='970' height='370' stroke-width='2.5px' rx='5px' />
            <text x='500' y='200' fill='white' text-anchor='middle' dominant-baseline='middle' font-size='84px'
                font-weight='bold'>
                Genomic SEM
            </text>
        </g>

         <g id='prs-tool' transform='translate(-325 -150)'>
            <rect x='0' y='0' width='650' height='300' stroke-width='5px' rx='20px' />
            <rect x='15' y='15' width='620' height='270' stroke-width='2.5px' rx='5px' />

            <text fill='white' font-size='50px'
                font-weight='bold'>
                <tspan x='325' y='100' text-anchor='middle' dominant-baseline='middle'>
                    Polygenic score
                </tspan>
                <tspan x='325' y='200' text-anchor='middle' dominant-baseline='middle'>
                    tool
                </tspan>
            </text>
        </g>

        <g id='arrow1' opacity='0.5'>
            <path d="M 0 0 l 17 17 l 45 0 l -17 -17 l 17 -17 l -45 0 l -17 17 z
            M 135 0 l 17 17 l -45 0 l -17 -17 l 17 -17 l 45 0 l -17 17 z" stroke-width='2px' fill='white'
                stroke-linejoin='round' />
        </g>

        <g id='arrow1-longer' opacity='0.5'>
            <path d="M 0 0 l 17 17 l 50 0 l -17 -17 l 17 -17 l -50 0 l -17 17 z
            M 150 0 l 17 17 l -50 0 l -17 -17 l 17 -17 l 50 0 l -17 17 z" stroke-width='2px' fill='white'
                stroke-linejoin='round' />
        </g>

        <g id='arrow1-longest' opacity='0.5'>
            <path d="M 0 0 l 17 17 l 60 0 l -17 -17 l 17 -17 l -60 0 l -17 17 z
            M 180 0 l 17 17 l -60 0 l -17 -17 l 17 -17 l 60 0 l -17 17 z" stroke-width='2px' fill='white'
                stroke-linejoin='round' />
        </g>

        <g id='arrow2' opacity='0.6'>
            <path d="M 0 0 l 40 40 l 100 0 l -40 -40 l 40 -40 l -100 0 l -40 40 z
            M 250 0 l 40 40 l -100 0 l -40 -40 l 40 -40 l 100 0 l -40 40 z
            M 400 0 l 40 40 l -100 0 l -40 -40 l 40 -40 l 100 0 l -40 40 z
            M 550 0 l 40 40 l -100 0 l -40 -40 l 40 -40 l 100 0 l -40 40 z" stroke-width='8px'
                stroke-linejoin='round' />
        </g>

        <g id='main-panel'>
            <g id='classic-gwas-panel' transform='translate(1000 800)'>
                <use href='#classic-gwas' x='0' y='0' />

                <text x='-200' y='-700' font-size='45px' fill='white' text-anchor='middle'>Phenotype</text>
                <use href='#pheno' x='-200' y='-500' />
                <use href='#arrow1' x='0' y='0' transform='translate(-200 -175) rotate(-90)'  />

                <text x='200' y='-700' font-size='45px' fill='white' text-anchor='middle'>Genotype</text>
                <use href='#geno' x='200' y='-500' />
                <use href='#arrow1' x='0' y='0' transform='translate(200 -175) rotate(-90)'  />

                <text x='1170' y='-200' font-size='45px' fill='white' text-anchor='middle'>
                    <tspan>Summary</tspan>
                    <tspan>statistics</tspan>
                </text>
                <use href='#sumstats' x='1170' y='0' />
                <use href='#arrow2' x='0' y='0' transform='translate(1170 0) rotate(180) translate(200)' />

            </g>
            <g id='meta-gwas-panel' transform='translate(2600 1400)'>
                <use href='#meta-gwas' x='0' y='0' />

                <use href='#sumstats' x='0' y='0' transform='translate(400 -575) scale(0.6)' opacity='0.6' />
                <use href='#arrow1-longest' x='0' y='0' transform='translate(400 -250) rotate(-90)' opacity='0.6'/>

                <use href='#sumstats' x='0' y='0' transform='translate(150 -575) scale(0.6)' opacity='0.6' />
                <use href='#arrow1-longest' x='0' y='0' transform='translate(150 -250) rotate(-90)' opacity='0.6'/>

                <use href='#sumstats' x='0' y='0' transform='translate(-100 -575) scale(0.6)' opacity='0.6' />
                <use href='#arrow1-longest' x='0' y='0' transform='translate(-100 -250) rotate(-90)' opacity='0.6'/>
                
                <!-- <use href='#sumstats' x='0' y='0' transform='translate(-425 -600)' opacity='0.6' /> -->
                <use href='#arrow1-longer' x='0' y='0' transform='translate(-425 -250) rotate(-90)' />

                <use href='#sumstats' x='-1375' y='0' />
                <use href='#arrow2' x='0' y='0' transform='translate(-1365 0) rotate(0) translate(200)' />
            </g>

            <g id='genomic-sem-panel' transform='translate(800 2000)'>
                <use href='#genomic-sem' x='0' y='0' />

                <use href='#sumstats' x='0' y='0' transform='translate(-400 -575) scale(0.6)' opacity='0.6' />
                <use href='#arrow1-longest' x='0' y='0' transform='translate(-400 -250) rotate(-90)' opacity='0.6'/>

                <use href='#sumstats' x='0' y='0' transform='translate(-150 -575) scale(0.6)' opacity='0.6' />
                <use href='#arrow1-longest' x='0' y='0' transform='translate(-150 -250) rotate(-90)' opacity='0.6'/>

                <use href='#sumstats' x='0' y='0' transform='translate(100 -575) scale(0.6)' opacity='0.6' />
                <use href='#arrow1-longest' x='0' y='0' transform='translate(100 -250) rotate(-90)' opacity='0.6' />
                
                <!-- <use href='#sumstats' x='0' y='0' transform='translate(425 -600)' opacity='0.6' /> -->
                <use href='#arrow1-longer' x='0' y='0' transform='translate(425 -250) rotate(-90)' />

                <use href='#sumstats' x='1375' y='0' />
                <use href='#arrow2' x='0' y='0' transform='translate(1365 0) rotate(180) translate(200)' />
            </g>

            <g id='prs-panel' transform='translate(2400 2500)'>
                <use href='#prs-tool' x='0' y='0' />

                <!-- <use href='#pheno' x='-220' y='-500' /> -->
                <use href='#arrow1' x='0' y='0' transform='translate(-220 -175) rotate(-90)'  />

                <use href='#geno' x='220' y='-500' />
                <use href='#arrow1' x='0' y='0' transform='translate(220 -175) rotate(-90)'  />

                <use href='#prs' x='1175' y='0' />
                <text x='1175' y='250' font-size='45px' fill='white' text-anchor='middle'>
                    <tspan>Polygenic</tspan>
                    <tspan>score</tspan>
                </text>
                <use href='#arrow2' x='-975' y='0' transform='rotate(180)'/>
            </g>
        </g>
        
        <pattern id='geno-pattern' x='0' y='0' width='1000' height='866' patternUnits='userSpaceOnUse'>
            <use href='#prs' fill='none' x='0' y='433' />
            <use href='#prs' fill='none' x='1000' y='433' />
            <use href='#geno' fill='none' x='250' y='0' />
            <use href='#geno' fill='none' x='250' y='866' />
            <use href='#sumstats' fill='none' x='750' y='0' />
            <use href='#sumstats' fill='none' x='750' y='866' />
            <use href='#pheno' fill='none' x='500' y='433' />
            <rect x='0' y='0' width='100%' height='100%' fill='none' stroke-width='3px' />            
        </pattern>

        <rect id='geno-wallpaper' x='0' y='0' width='100%' height='100%' fill='url(#geno-pattern)' stroke-width='3px' />
        
    </defs>

    <use href='#main-panel' fill='none'  stroke='white' filter='url(#soft-glow)' />
    <rect x='0' y='0' width='100%' height='100%' fill='none' stroke-width='3px' />
</svg><!-- 
Author: Ivan Voronin
https://github.com/IvanVoronin

Feb 24, 2022
License: CC BY-NC-SA 4.0 https://creativecommons.org/licenses/by-nc-sa/4.0/ 
-->
<svg width='100px' height="100px" viewBox="0 0 100 100"
style='stroke:white;text-color:white;fill:none;opacity:0.5;'>
<defs>
    <mask id='list-mask'>
        <rect width='100%' height='100%' fill='white'/>
        <g id='list-item' transform='translate(0 24.5)'>
            <circle cx='25' cy='0' r='5' stroke='black' stroke-width='3'/>
            <line x1='37' y1='0' x2='75' y2='0' stroke='black' stroke-width='3' stroke-linecap='round'/>
        </g>
        <use href='#list-item' y='17' />
        <use href='#list-item' y='34' />
        <use href='#list-item' y='51' />
    </mask>

    <rect id='list-icon' x='10' y='10' width='80' height='80' stroke-width='5' rx='10' fill='white' 
mask='url(#list-mask)'/>
</defs>
</svg><!-- 
Author: Ivan Voronin
https://github.com/IvanVoronin

Feb 24, 2022
License: CC BY-NC-SA 4.0 https://creativecommons.org/licenses/by-nc-sa/4.0/ 
-->
<svg width='100px' height="100px" viewBox="0 0 100 100"
style='stroke:white;text-color:white;fill:none;opacity:0.5;'>
<defs>
    <mask id='ab-mask'>
        <rect width='100%' height='100%' fill='white'/>
<text font-size='46' stroke='black' dominant-baseline='middle' fill='black' stroke-linejoin='round'
font-weight='bold'>
    <tspan x='16' y='40'>A</tspan>
    <tspan x='56' y='67'>Z</tspan>
</text>
    </mask>
    
    <rect id='glossary-icon' x='10' y='10' width='80' height='80' stroke-width='5' rx='10' fill='white' 
mask='url(#ab-mask)'/>
</defs>
</svg>
&lt;/div&gt;

&lt;div id='navig-links' class='navig-links-class'&gt;
&lt;a href='#contents' style='width:48%;float:left;'&gt;
&lt;svg viewBox='10 10 80 80'&gt;
    &lt;use href='#list-icon'&gt;&lt;/use&gt;
&lt;svg&gt;
&lt;/a&gt;
&lt;a href='#glossary' style='width:48%;float:right;'&gt;
&lt;svg viewBox='10 10 80 80'&gt;
    &lt;use href='#glossary-icon'&gt;&lt;/use&gt;
&lt;svg&gt;
&lt;/a&gt;
&lt;/div&gt;

&lt;h1&gt;Mining the genome&lt;/h1&gt;
&lt;h2 style='text-align:center;margin-right:auto;margin-left:auto;width:50%;'&gt;
&lt;svg viewBox="0 0 1500 500" filter='url(#neon-glow)'
    style='stroke:white;text-color:white;fill:none;'&gt;
    &lt;use href='#geno' transform='translate(250 250)'&gt;&lt;/use&gt;
    &lt;use href='#sumstats' transform='translate(750 250)'&gt;&lt;/use&gt;
    &lt;use href='#prs' transform='translate(1250 250)'&gt;&lt;/use&gt;
&lt;/svg&gt;
&lt;/h2&gt;

&lt;div class='title-label'&gt;
&lt;p style='margin-bottom:15px;'&gt;
Ivan Voronin, PhD&lt;br&gt;
Université Laval
&lt;/p&gt;
&lt;p style='font-size:0.8em;opacity:0.7;'&gt;
Behavioral Genetics Journal Club&lt;br&gt;
presented on January 27, 2022
&lt;/p&gt;

&lt;!-- TODO: Указать альтернативные шрифты (в т.ч. в СВГ) --&gt;



---
name: contents

# Index

&lt;!--
Гетерогенность GWAS: возраст, м/б пол

Уточнение про Genomic SEM - в ходе Genomic SEM не вычисляется полигенный индекс, это GWAS

Количественный характер полигенного индекса - это всегда количественная характеристика,
даже если признак дискретный
Полигенный индекс для полигенных признаков
--&gt;

&lt;div class='big-text'&gt;
&lt;ol&gt;
&lt;li&gt;&lt;a href='#first-panel'&gt;Phenotype and genome, gene, genotype&lt;/a&gt;

&lt;li&gt;&lt;a href='#second-panel'&gt;Classic GWAS and GWAS summary statistics&lt;/a&gt;

&lt;li&gt;&lt;a href='#third-panel'&gt;Meta-GWAS and Genomic SEM&lt;/a&gt;

&lt;li&gt;&lt;a href='#fourth-panel'&gt;Polygenic score&lt;/a&gt;

&lt;li&gt;&lt;a href='#glossary'&gt;Glossary and abbreviations&lt;/a&gt;
&lt;/ol&gt;
&lt;/div&gt;

---
name: first-panel
class: main-panel

&lt;div class='main-panel-svg'&gt;
&lt;svg viewBox="0 -50 4000 2850"
    style='stroke:white;text-color:white;fill:none;'&gt;
    &lt;g class='active-panel' transform='translate(1000 800)'&gt;
        &lt;text x='-200' y='-700' font-size='45px' fill='white' text-anchor='middle'&gt;Phenotype&lt;/text&gt;
        &lt;use href='#pheno' x='-200' y='-500' /&gt;
        &lt;text x='200' y='-700' font-size='45px' fill='white' text-anchor='middle'&gt;Genotype&lt;/text&gt;
        &lt;use href='#geno' x='200' y='-500' /&gt;
    &lt;/g&gt;
    &lt;g class='inactive-panel' transform='translate(1000 800)'&gt;
        &lt;use href='#classic-gwas' x='0' y='0' /&gt;
        &lt;use href='#arrow1' x='0' y='0' transform='translate(-200 -175) rotate(-90)'  /&gt;
        &lt;use href='#arrow1' x='0' y='0' transform='translate(200 -175) rotate(-90)'  /&gt;
        &lt;text x='1170' y='-200' font-size='45px' fill='white' text-anchor='middle'&gt;
            &lt;tspan&gt;Summary&lt;/tspan&gt;
            &lt;tspan&gt;statistics&lt;/tspan&gt;
        &lt;/text&gt;
        &lt;use href='#sumstats' x='1170' y='0' /&gt;
        &lt;use href='#arrow2' x='0' y='0' transform='translate(1170 0) rotate(180) translate(200)' /&gt;
    &lt;/g&gt;
    &lt;use href='#meta-gwas-panel' class='inactive-panel' /&gt;
    &lt;use href='#genomic-sem-panel' class='inactive-panel' /&gt;
    &lt;use href='#prs-panel' class='inactive-panel' /&gt;
&lt;/svg&gt;
&lt;/div&gt;


&lt;!--
&lt;script&gt;
    $(document).ready(function() {
        $('#pheno-inst').glowingSign({
            placement: 'bottom',
            content:
                '&lt;ul&gt;' + 
                '&lt;li&gt;Any measurable human trait&lt;/li&gt;' +
                '&lt;li&gt;Continuous or discrete&lt;/li&gt;' +
                '&lt;li&gt;Variability and reliability are very important&lt;/li&gt;' +
                '&lt;/ul&gt;'     
        })
    });
&lt;/script&gt;
--&gt;

---
class:two-columns

# Phenotype


.left-col.pad-right-020[
Phenotype is a measurable trait

.indent[
Cognitive: &lt;br&gt;
cognitive ability, reading skills

Behavioral: &lt;br&gt;
smoking, risk-taking

Psychopathological:&lt;br&gt;
depression, schizophrenia

Developmental:&lt;br&gt;
ADHD, autism

Socioeconomic: &lt;br&gt;level of education, income
]

Traits can be continuous or discrete.
Discrete traits are usually a diagnostic status of disorder

]

.right-col.center[
&lt;img src="pics/pheno-1.png" width="80%" /&gt;
&lt;img src="pics/pheno-2.png" width="80%" /&gt;




]

---
class: two-columns
# Genome

.left-col.center[

&lt;div id="dna-carousel" class="my-carousel"&gt;
    &lt;div class="my-carousel-item"  style='display:none;'&gt;
        &lt;img src="pics/human_karyotype.jpg" 
            alt="Human karyotype" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://commons.wikimedia.org/wiki/File:Karyotype_(normal).jpg'&gt;
            Wikimedia
        &lt;/a&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='margin-right:7%;margin-left:7%;'&gt;
        &lt;img src="pics/dna_structure.png" 
            alt="DNA structure"
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link' 
           href='https://commons.wikimedia.org/wiki/File:DNA_Structure%2BKey%2BLabelled.pn_NoBB.png'&gt;
            Wikimedia
        &lt;/a&gt;
    &lt;/div&gt;
  &lt;/div&gt;
&lt;/div&gt;

]

.right-col.pad-left-020[
Genome is complete genetic information of an organism stored in the DNA

DNA is a sequence of base pairs, 
or the pairs of complementary linked nucleotides:&lt;br&gt;
&lt;p class='pad-left-50'&gt;
&lt;em&gt;A&lt;/em&gt;denine &amp;mdash; &lt;em&gt;T&lt;/em&gt;hymine &lt;br&gt;
&lt;em&gt;G&lt;/em&gt;uanine &amp;mdash; &lt;em&gt;C&lt;/em&gt;ytosine
&lt;/p&gt;

Human genome (HG) is diploid: one copy of the genome comes from the mother and one from the father

3.1B base pairs of the HG are typically stored in 23 pairs of chromosomes:
44 autosomes and 2 sex chromosomes

]

---
class:two-columns

# Gene

.left-col.center[

&lt;div class="my-carousel" style='position:relative;margin-left:5%;margin-right:5%;'&gt;
        &lt;img src="pics/Eukaryotic_Transcription.gif" 
            alt="Eukaryotic transcription" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://commons.wikimedia.org/wiki/File:Eukaryotic_Transcription.gif'&gt;
            Wikimedia
        &lt;/a&gt;
  &lt;/div&gt;
&lt;/div&gt;

]

.right-col.pad-left-020[
Gene is a region of DNA that encodes a biologically active molecule. 
It is either a protein or RNA

Total number of genes in the HG is over 46,000, of 
which ~20,000 are protein-coding genes (1.5% of HG)&lt;br&gt;

Expression of a gene is the process of building a protein/RNA 
product from the blueprint of this gene. Gene expression 
is intricately regulated

The profiles of gene expression vary across the cells of the body 
depending on the type of tissue and cell's life stage

]

---
class:two-columns

# Genetic variation

.left-col.center[
&lt;div class="my-carousel"&gt;
    &lt;div class="my-carousel-item"&gt;
        &lt;img src="pics/ideogram_grch38p4.png" 
            alt="GRCh38p4 ideogram" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
        href='https://www.sanger.ac.uk/data/genome-reference-consortium/'&gt;
            Genome Reference Consortium
        &lt;/a&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item"  style='display:none;'&gt;
        &lt;img src="pics/genetic-variation.jpg" 
            alt="Common types of genetic variants" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
        href='https://www.frontiersin.org/articles/10.3389/fbioe.2015.00013/full'&gt;
            Cardoso et al. (2015)
        &lt;/a&gt;
    &lt;/div&gt;
&lt;/div&gt;
]

.right-col.pad-left-020[

Genetic variants (polymorphisms) are located on the reference genome.
Reference genome is an idealized nucleotide seqience of a single person

A real HG typically differs from the reference genome in 20M base pairs

The [dbSNP](https://www.ncbi.nlm.nih.gov/snp/) catalogue lists over 900M polymorphisms. About 15M of them occur in at least 
1% of human population

Some types of genetic variance:
single nucleotide polymorphism, insertion, deletion, 
copy-number variation,
large-scale structural variation

]

---
class:two-columns

# SNP

.left-col.center.align-to-the-top[
&lt;h3 style='margin-bottom:15pt;margin-top:0;'&gt;
&lt;a href='https://www.ncbi.nlm.nih.gov/snp/rs1222063'&gt;
    dbSNP: rs1222063
&lt;/a&gt;
&lt;/h3&gt;

&lt;div class="my-carousel" style='margin-left:10%;margin-right:10%;'&gt;
    &lt;div class="my-carousel-item"&gt;
        &lt;img src="pics/dbSNP1.png" 
            alt="Example of SNP location" 
            width='100%'
        /&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='display:none;'&gt;
        &lt;img src="pics/dbSNP2.png" 
            alt="Example of SNP: allele frequency"
            width='100%'
        /&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='display:none;'&gt;
        &lt;img src="pics/dbSNP3.png" 
            alt="Example of SNP: the view of genomic region"
            width='100%'
        /&gt;
    &lt;/div&gt;
&lt;/div&gt;

]

.right-col.pad-left-020[

Single nucleotide polymorphism (SNP)
is the most common type of genetic variation

.indent[
SNP identification: &lt;br&gt;
Variant ID (rsID): rs11420276 &lt;br&gt;
Chromosome: 1 &lt;br&gt;
Position: 96,136,884 (GRCh38) &lt;br&gt;
Alleles: G&gt;A
]

Allele is an alternative form of a specific genetic variant. Allele frequencies vary a lot between populations

Linkage disequilibrium (LD) is a nonrandom association 
between two SNPs that are close in genome
]

---
class:two-columns-and-big-display

# Genotype

.left-col.pad-right-020[
Genotype is a combination of two alleles carried by 
an individual at a specific locus. (It is also the 
combination of alleles across all considered loci)
]

.right-col.pad-left-020[
In each locus, one allele is picked as the effect allele and scored with 1, the other is scored with 0. The sum of allelic scores gives a genotype score
]

.big-display.table-container[
&lt;table class="table" style="margin-left: auto; margin-right: auto;"&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style="text-align:center;"&gt; CHR &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; POS &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; rsID &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; A1 &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; A2 &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; Freq(A1) &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; Genotype 1 &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; Genotype 2 &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; 10 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 12,950,881 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; rs72775679 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; A &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; G &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.01 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; AA=0 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; GG=2 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; 10 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 12,950,954 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; rs10795993 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; A &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; T &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.24 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; AT=1 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; AT=1 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; 10 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 12,951,056 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; rs12220635 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; G &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; A &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.11 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; AA=2 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; AG=1 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; 10 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 12,951,508 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; rs7899716 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; T &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; G &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.11 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; GG=2 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; TG=1 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; 10 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 12,951,727 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; rs12243442 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; A &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; G &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.11 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; GG=2 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; AG=1 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;
]

---
class:two-columns

# Genotyping

.left-col.center[
&lt;div class="my-carousel"&gt;
    &lt;div class="my-carousel-item"  style='display:none;'&gt;
        &lt;img src="pics/illumina-microarray.jpg" 
            alt="How do Illumina Microarrays Work?" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://www.illumina.com/science/technology/microarray.html'&gt;
            Illumina
        &lt;/a&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='margin-left:5%;margin-right:5%;'&gt;
        &lt;img src="pics/genotype-chips.jpg" 
            alt="DNA structure"
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link' 
           href='https://www.thermofisher.com/order/catalog/product/901615'&gt;
            ThermoFisher
        &lt;/a&gt;, 
        &lt;a class='source-link' 
           href='https://www.illumina.com/products/by-type/microarray-kits/infinium-psycharray.html'&gt;
            Illumina
        &lt;/a&gt;
    &lt;/div&gt;
  &lt;/div&gt;
&lt;/div&gt;
]

.right-col.pad-left-020[
Genotyping refers to the measurement of genotypes from the DNA that 
has been extracted from biosamples (saliva, blood)

Genotyping happens on a microarray chip. 
Each cell of the chip lights up when in contact with a DNA sample that carries specific allele.
The intensity of signal indicates which allele is present in the sample

Illumina PsychArray-24 measures ~600K SNPs

The genotype data passes through rigorous quality control and imputation
before being used in a GWAS
]

---
name: second-panel
class: main-panel

&lt;div class='main-panel-svg'&gt;
&lt;svg viewBox="0 -50 4000 2850" font-family='Noto Sans'
    style='stroke:white;text-color:white;fill:none;'&gt;
    &lt;g class='active-panel'&gt;
        &lt;use href='#classic-gwas-panel' /&gt;
    &lt;/g&gt;
    &lt;use href='#meta-gwas-panel' class='inactive-panel' /&gt;
    &lt;use href='#genomic-sem-panel' class='inactive-panel' /&gt;
    &lt;use href='#prs-panel' class='inactive-panel' /&gt;
&lt;/svg&gt;
&lt;/div&gt;


---
class:two-columns
# Classic GWAS

.left-col.pad-right-020[
Genome-wide association study (GWAS) is a study
of associations between a phenotype and genetic variants across the whole genome

Additive model of genetic effect assumes that 
each instance of effect allele independently adds to the trait.
Therefore, in a GWAS, a phenotype value is regressed on a genotype score

The associations are estimated and tested using either simple or logistic regression,
depending on whether the phenotype is continuous or discrete

The size of association can be expressed as beta, Z, or odds ratio (OR)

]

.right-col.center[
&lt;div class="my-carousel" style='padding-left:5%;padding-right:5%;'&gt;
        &lt;img src="pics/gwas-one-snp.jpg" 
            alt="How GWAS works" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://www.frontiersin.org/articles/10.3389/fgene.2020.00243/full'&gt;
            Ye et al. (2020)
        &lt;/a&gt;
&lt;/div&gt;
]

---
class:two-columns-and-big-display

# GWAS summary statistics

.left-col.pad-right-020[
GWAS summary statistics are the regression results from GWAS.
They are stored in a table
that lists each SNP's ID,
effect size and statistical significance
]

&lt;div class="my-carousel" 
style='grid-area:2/1/4/3;align-self:start;margin-top:15px;' &gt;
&lt;div class="my-carousel-item"
style='display:grid;grid-template-rows:125px auto;grid-template-cols:1fr 1fr;'&gt;
&lt;div class='pad-left-020' style='grid-row:1/2;grid-column:2/3;align-self:end;'&gt;

&lt;h3 style='font-size:120%;'&gt;
&lt;a href='https://www.nature.com/articles/s41588-018-0147-3'&gt;
    EA GWAS, Lee et al. (2018)    
&lt;/a&gt;
&lt;/h3&gt;

10,101,243 SNPs

&lt;/div&gt;

&lt;div class='table-container' style='grid-row:2/3;grid-column:1/3;'&gt;

&lt;table class="table" style="margin-left: auto; margin-right: auto;"&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style="text-align:center;"&gt; MarkerName &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; CHR &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; POS &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; A1 &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; A2 &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; EAF &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; Beta &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; SE &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; Pval &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; rs13090388 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 3 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 49,391,082 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; C &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; T &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.6905 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; -0.02852 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.00184 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 4.29e-54 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; rs7630869 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 3 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 49,522,543 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; C &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; T &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.6922 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; -0.02848 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.00184 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 4.61e-54 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; rs7623659 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 3 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 49,414,791 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; T &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; C &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.3095 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.02847 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.00184 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 4.75e-54 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; rs11922013 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 3 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 49,458,355 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; G &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; C &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.6905 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; -0.02844 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.00184 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 5.94e-54 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;

&lt;/div&gt;
&lt;/div&gt;

&lt;div class="my-carousel-item"
style='display:none;grid-template-rows:125px auto;grid-template-cols:1fr 1fr;align-self:start;'&gt;
&lt;div style='grid-row:1/2;grid-column:2/3;align-self:end;'&gt;

&lt;h3 style='font-size:120%;'&gt;
&lt;a href='https://www.nature.com/articles/s41588-018-0269-7'&gt;
    ADHD GWAS, Demontis et al. (2019)    
&lt;/a&gt;
&lt;/h3&gt;

8,094,095 SNPs
&lt;/div&gt;

&lt;div class='table-container' style='grid-row:2/3;grid-column:1/3;'&gt;
&lt;table class="table" style="margin-left: auto; margin-right: auto;"&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style="text-align:center;"&gt; CHR &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; SNP &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; BP &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; A1 &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; A2 &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; FRQ_A_19099 &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; FRQ_U_34194 &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; INFO &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; OR &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; SE &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; P &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; 8 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; rs17396518 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 108,690,829 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; T &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; G &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.557 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.561 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.993 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.99243 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.0140 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.5849 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; 8 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; rs983166 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 108,681,675 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; A &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; C &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.565 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.567 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.996 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.99890 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.0136 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.9365 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; 8 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; rs28842593 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 103,044,620 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; T &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; C &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.841 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.841 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.885 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.99402 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.0221 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.7878 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; 8 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; rs35107696 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 109,712,249 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; A &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; AT &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.771 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.771 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.996 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 1.00180 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.0160 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.9112 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; ... &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;

&lt;/div&gt;
&lt;/div&gt;
&lt;/div&gt;

---
class:two-columns

# GWAS hits

.left-col.pad-right-020[
GWAS hit is a SNP effect that reached the level of
statistical significance in GWAS

To be regarded as a GWAS hit, the SNP effect must pass
a stringent p-threshold (e.g., `\(p &lt; 5 \times 10^{-8}\)`).
This is necessary because when 5M SNPs are tested,
we would expect 250K false positive hits at `\(p &lt; 0.05\)`

Often, GWAS hits do not replicate across different GWASes
of the same trait

The distribution of p-values, incl. GWAS hits, is usually presented
in a Manhattan plot

&lt;a href='https://www.ebi.ac.uk/gwas/home'&gt;GWAS Catalog&lt;/a&gt; lists published GWAS hits
]

.right-col.center[
&lt;div class="my-carousel"&gt;
        &lt;img src="pics/adhd-manhattan.jpg" 
            alt="An example of manhattan plot"
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link' 
           href='https://www.nature.com/articles/s41588-018-0269-7'&gt;
            Demontis et al. (2018)
        &lt;/a&gt;
&lt;/div&gt;
]

---
class: two-columns

# Results of GWAS

.right-col.pad-left-020[
    &lt;!-- An example of GWAS and everything that we get from it --&gt;
Usually, GWAS also reports:
- SNP heritability &amp;mdash; the amount of the trait's variability explained by all SNPs
- genetic correlations &amp;mdash; the overlap between genetic variance of different traits
- biological annotation of associations &amp;mdash; what is known about 
the function of genomic regions where GWAS hits have been discovered
- polygenic score analysis &amp;mdash; how well the aggregate of discovered associations predicts the trait in an independent sample 

]

.left-col.center[
&lt;div class="my-carousel"&gt;
    &lt;div class="my-carousel-item"&gt;
        &lt;img src="pics/ADHD_GWAS_gencor.png" 
            alt="An example of genetic correlations from ADHD GWAS (Demontis et al., 2018)" 
            width='100%'
        /&gt;        
        &lt;a class='source-link' 
           href='https://www.nature.com/articles/s41588-018-0269-7'&gt;
            Demontis et al. (2018)
        &lt;/a&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='display:none;'&gt;
        &lt;img src="pics/EA_GWAS_bioannot.jpg" 
            alt="An example of biological annotation from EA GWAS (Lee et al., 2018)" 
            width='100%'
        /&gt;        
        &lt;a class='source-link'
            href='https://www.nature.com/articles/s41588-018-0147-3'&gt;
            EA GWAS, Lee et al. (2018)    
        &lt;/a&gt;
    &lt;/div&gt;
&lt;/div&gt;
]



---
name: third-panel
class: main-panel

&lt;div class='main-panel-svg'&gt;
&lt;svg id='second-panel' viewBox="0 -50 4000 2850"
    style='stroke:white;text-color:white;fill:none;'&gt;
    &lt;g class='active-panel'&gt;
        &lt;use href='#classic-gwas-panel' /&gt;
        &lt;use href='#meta-gwas-panel' /&gt;
        &lt;use href='#genomic-sem-panel' /&gt;
    &lt;/g&gt;
    &lt;use href='#prs-panel' class='inactive-panel' /&gt;
&lt;/svg&gt;
&lt;/div&gt;


---
class:two-columns

# Meta-GWAS

.left-col.pad-right-020.align-to-the-top[
Meta-analysis of GWA datasets (meta-GWAS) combines the effect sizes from multiple studies
to achieve more precise estimates of SNP effects. 
The input data of meta-GWAS are GWAS summary statistics

Most modern GWASes are meta-GWAS

&lt;div class="my-carousel"&gt;
    &lt;div class="my-carousel-item"&gt;
Advantages of meta-GWAS:
&lt;ul&gt;
&lt;li&gt;higher statistical power than classic GWAS
&lt;li&gt;no need assemble all genotype data in one place
&lt;li&gt;easier to add new data to an existing GWAS
&lt;/ul&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='display:none;'&gt;
Disadvantages of meta-GWAS:
&lt;ul&gt;
&lt;li&gt;sumstats must be produced from the clean genotype data
and harmonized phenotype; 
&lt;li&gt;the sub-samples must not overlap, participants must be unrelated
&lt;/ul&gt;
    &lt;/div&gt;
&lt;/div&gt;

]

.right-col.center[
&lt;div class="my-carousel"&gt;
    &lt;div class="my-carousel-item" style='margin-left:7%;margin-right:7%;'&gt;
        &lt;img src="pics/forest-plot.jpg" 
            alt="Forest plot" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://ard.bmj.com/content/73/12/2130'&gt;
            Evangelou et al. (2014)
        &lt;/a&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='display:none;transform:rotate(-90deg);text-align:center;'&gt;
        &lt;img src="pics/EA1_Rietveld_2013_forest.png" 
             alt="Forest plot from EA1 GWAS, Rietveld et al. (2013)" 
             width='70%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3751588/'&gt;
            Rietveld et al. (2013)
        &lt;/a&gt;
    &lt;/div&gt;
&lt;/div&gt;

]

---

# Meta-GWAS for EA

&lt;div style='display:flex;flex-flow:row wrap;justify-content:space-between;align-content:space-between;gap:20px 5px;'&gt;

&lt;div style='height:230px;'&gt;
    &lt;a href='https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3751588/'&gt;
    EA1 GWAS, 2013:&lt;/a&gt; N = 125K, 3 GWAS hits&lt;br&gt;
    &lt;img src="pics/EA1_Rietveld_2013_manhattan.png" 
        alt="Manhattan plot from EA1 GWAS (Rietveld et al. 2013)"
        height='170px'
        style='margin-top:10pt;'
    /&gt;
&lt;/div&gt;

&lt;div style='height:230px;'&gt;
    &lt;a href='https://pubmed.ncbi.nlm.nih.gov/27225129/'&gt;
        EA2 GWAS, 2016:&lt;/a&gt; N = 293K, 74 GWAS hits&lt;br&gt;
    &lt;img src="pics/EA2_Okbay_2016_manhattan.jpg" 
        alt="Manhattan plot from EA2 GWAS (Okbay et al. 2016)"
        height='170px'        
        style='margin-top:10pt;'
    /&gt;
&lt;/div&gt;

&lt;div style='height:250px;'&gt;
    &lt;a href='https://www.nature.com/articles/s41588-018-0147-3'&gt;
        EA3 GWAS, 2018:&lt;/a&gt; N = 1.1M, 1,271 GWAS hits&lt;br&gt;
    &lt;img src="pics/EA3_Lee_2018_manhattan.jpg" 
        alt="Manhattan plot from EA3 GWAS (Lee et al. 2018)"
        height = "220px"
        style='margin-top:10pt;'
    /&gt;
&lt;/div&gt;

&lt;div style='height:250px'&gt;
    &lt;a href='https://twitter.com/dr_appie/status/1493648652115529731'&gt;
        EA4 GWAS, pending:&lt;/a&gt; N = 3M&lt;br&gt;
    &lt;img src="pics/EA4_unpub_manhattan.jpeg" 
        alt="Manhattan plot from EA4 GWAS (under review)"
        height = "220px"
        style = "margin-top:10pt;background-color:#e7e6ec;"
    /&gt;
&lt;/div&gt;

&lt;/div&gt;

---
class:two-columns

# Genomic SEM

.left-col.pad-right-020[

Genomic SEM tests and estimates a model of genetic liability shared by several traits

SEM model describes the links between latent genetic factors
and genetic risks previously identified in GWASes for specific traits

.indent[
Step 1: Genetic covariance matrix is estimated 
from GWAS summary statistics

Step 2: Paremeters of the model are
estimated
]

Then, it is possible to run a GWAS for the latent genetic factor
by adding each SNP to the model and estimating its effect on the factor

]

.right-col.center[

&lt;div class="my-carousel"&gt;
    &lt;div class="my-carousel-item" style='margin-left:10%;margin-right:10%;'&gt;
        &lt;img src="pics/ExtGWAS_gen_cor.jpg" 
            alt="Genetic correlations from ExtGWAS, Karlsson-Linner et al. (2021)" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://www.nature.com/articles/s41593-021-00908-3'&gt;
            Karlsson Linner et al. (2021)
        &lt;/a&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='display:none;margin-left:7%;margin-right:7%;'&gt;
        &lt;img src="pics/ExtGWAS_path_diagram.jpg" 
             alt="Path diagram from ExtGWAS, Karlsson-Linner et al. (2021)" 
             width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://www.nature.com/articles/s41593-021-00908-3'&gt;
            Karlsson Linner et al. (2021)
        &lt;/a&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='display:none;margin-left:12%;margin-right:12%;'&gt;
        &lt;img src="pics/ExtGWAS.png" 
             alt="Path diagram from ExtGWAS, Karlsson-Linner et al. (2021)" 
             width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://www.nature.com/articles/s41593-021-00908-3'&gt;
            Karlsson Linner et al. (2021)
        &lt;/a&gt;
    &lt;/div&gt;
&lt;/div&gt;

]

---
class: two-columns

# Externalizing GWAS

.left-col.pad-right-020[
Application of Genomic SEM:&lt;br&gt; 
&lt;a href='https://www.nature.com/articles/s41593-021-00908-3'&gt;GWAS for externalizing phenotype,&lt;/a&gt;
or behaviors and disorders related to the lack of self-control

Included GWAS data:&lt;br&gt;
ADHD; smoking; cannabis use; problematic alcohol use;
risk tolerance; age at first sex; number of sexual partners

N ~ 1.5M, 579 GWAS hits

The study produced GWAS summary statistics that
are available for further study of
externalizing syndrom

]

.right-col.center[

&lt;div class="my-carousel"&gt;
    &lt;div class="my-carousel-item" style='margin-left:7%;margin-right:7%;'&gt;
        &lt;img src="pics/ExtGWAS_path_diagram.jpg" 
             alt="Path diagram from ExtGWAS, Karlsson-Linner et al. (2021)" 
             width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://www.nature.com/articles/s41593-021-00908-3'&gt;
            Karlsson Linner et al. (2021)
        &lt;/a&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='display:none;'&gt;
        &lt;img src="pics/ExtGWAS_manhattan.jpg" 
             alt="Manhattan plot from ExtGWAS, Karlsson-Linner et al. (2021)" 
             width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://www.nature.com/articles/s41593-021-00908-3'&gt;
            Karlsson Linner et al. (2021)
        &lt;/a&gt;
    &lt;/div&gt;
&lt;/div&gt;

]


---
class:two-columns

# Non-cog GWAS

.left-col.pad-right-020[
Application of Genomic SEM:&lt;br&gt;
&lt;a href='https://www.nature.com/articles/s41588-020-00754-2'&gt;Non-cognitive GWAS&lt;/a&gt;&lt;br&gt;
(GWAS-by-subtraction) 

Genetic variance of EA was modeled as
a sum of the variance related to cognitive ability
and the variance independent of cognitive ability, 
or non-cognitive variance

The summary statistics from GWAS for EA and cognitive
ability were used as input data

N=510K, 157 GWAS hits

The GWAS summary statistics for cognitive and non-cognitive
components of EA were published
]

.right-col.center[
&lt;div class="my-carousel"&gt;
    &lt;div class="my-carousel-item" style='margin-left:10%;margin-right:10%;'&gt;
        &lt;img src="pics/non-cog-GWAS.png"
            alt="GWAS for non-cognitive variance of educational attainment (Demange et al., 2021)" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
            href='https://www.nature.com/articles/s41588-020-00754-2'&gt;
            Demange et al. (2021)
        &lt;/a&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='display:none;'&gt;
        &lt;img src="pics/non-cog-GWAS-gencor.png"
            alt="Genetic correlations from the GWAS for non-cognitive variance of educational attainment (Demange et al., 2021)" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
            href='https://www.nature.com/articles/s41588-020-00754-2'&gt;
            Demange et al. (2021)
        &lt;/a&gt;
    &lt;/div&gt;
&lt;/div&gt;
]

---
name: fourth-panel
class: main-panel

&lt;div class='main-panel-svg'&gt;
&lt;svg id='third-panel' viewBox="0 -50 4000 2850" font-family='Noto Sans'
    style='stroke:white;text-color:white;fill:none;'&gt;
    &lt;g class='active-panel'&gt;
        &lt;use href='#classic-gwas-panel' /&gt;
        &lt;use href='#meta-gwas-panel' /&gt;
        &lt;use href='#genomic-sem-panel' /&gt;
        &lt;use href='#prs-panel' /&gt;
    &lt;/g&gt;
&lt;/svg&gt;
&lt;/div&gt;

---
class:two-columns

# Polygenic score

.right-col.align-to-the-top[
&lt;div style='width:100%;margin:auto;margin-bottom:40px;font-size:80%;'&gt;
.table-container[
&lt;table class="table" style="margin-left: auto; margin-right: auto;"&gt;
&lt;thead&gt;&lt;tr&gt;
&lt;th style="border-bottom:hidden;padding-bottom:0; padding-left:0px;padding-right:0px;text-align: center; " colspan="1"&gt;&lt;div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; "&gt;ID&lt;/div&gt;&lt;/th&gt;
&lt;th style="border-bottom:hidden;padding-bottom:0; padding-left:0px;padding-right:0px;text-align: center; " colspan="1"&gt;&lt;div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; "&gt;alleles&lt;/div&gt;&lt;/th&gt;
&lt;th style="border-bottom:hidden;padding-bottom:0; padding-left:0px;padding-right:0px;text-align: center; " colspan="1"&gt;&lt;div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; "&gt;beta&lt;/div&gt;&lt;/th&gt;
&lt;th style="border-bottom:hidden;padding-bottom:0; padding-left:0px;padding-right:0px;text-align: center; " colspan="3"&gt;&lt;div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; "&gt;genotype 1&lt;/div&gt;&lt;/th&gt;
&lt;th style="border-bottom:hidden;padding-bottom:0; padding-left:0px;padding-right:0px;text-align: center; " colspan="3"&gt;&lt;div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; "&gt;genotype 2&lt;/div&gt;&lt;/th&gt;
&lt;/tr&gt;&lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; rs1 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; A &amp;gt; G &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.20 &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; GG &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0 &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 0.00 &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; AG &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 1 &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 0.20 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; rs2 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; A &amp;gt; C &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.15 &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; AC &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 1 &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 0.15 &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; CC &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0 &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 0.00 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; rs3 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; T &amp;gt; G &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.32 &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; TT &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 2 &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 0.64 &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; TG &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 1 &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 0.32 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; rs4 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; C &amp;gt; A &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.27 &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; AC &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 1 &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 0.27 &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; AC &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 1 &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 0.27 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; rs5 &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; G &amp;gt; T &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 0.05 &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; GT &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 1 &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 0.05 &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; GG &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 2 &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; 0.10 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;font-weight: bold;border-bottom-width:0;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:center;font-weight: bold;border-bottom-width:0;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:center;font-weight: bold;border-bottom-width:0;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:right;font-weight: bold;border-bottom-width:0;"&gt; PRS 1 &lt;/td&gt;
   &lt;td style="text-align:center;font-weight: bold;border-bottom-width:0;"&gt; = &lt;/td&gt;
   &lt;td style="text-align:left;font-weight: bold;border-bottom-width:0;"&gt; 1.11 &lt;/td&gt;
   &lt;td style="text-align:right;font-weight: bold;border-bottom-width:0;"&gt; PRS 2 &lt;/td&gt;
   &lt;td style="text-align:center;font-weight: bold;border-bottom-width:0;"&gt; = &lt;/td&gt;
   &lt;td style="text-align:left;font-weight: bold;border-bottom-width:0;"&gt; 0.89 &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;
]

&lt;/div&gt;

.pad-left-020[
    To compute a polygenic score, 
    multiply the SNP's effect statistic from a GWAS
    by the person's genotype score at this locus. 
    Then, sum up the product across the SNPs. 
    Polygenic score is a continuous measure
]
]

.left-col.pad-right-020.align-to-the-top[
Polygenic score is an index of genetic risk or propensity for a trait calculated for a specific genotype on the basis of the known links between this trait and genetic variants

Different terms and abbreviations are used for polygenic score

&lt;div class="my-carousel center" style='margin-left:15%;margin-right:15%;'&gt;
        &lt;img src="pics/prs.jpg" 
            alt="Abbreviations of PRS" 
            width='100%'
            style = "margin-bottom:10px;"
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://mobile.twitter.com/RoshniAPatel/status/1263524501750337536'&gt;
            twitter@RoshniAPatel
        &lt;/a&gt;
&lt;/div&gt;

]

---
class:two-columns

# Performance of polygenic score

.left-col.pad-right-020[
`\(R^2\)` is a percent of the variance of a trait
explained by a polygenic score

`\(R^2\)` depends on: &lt;br&gt;
heritability of the trait; 
the accuracy of GWAS summary statistics;
the match between the ancestry of
GWAS and target populations;
the match between the GWAS and target phenotype

It is also common to compare the levels of a trait
in the groups with high and low polygenic score values
]

.right-col.center[
&lt;div class="my-carousel center"&gt;
    &lt;div class="my-carousel-item"&gt;
        &lt;img src="pics/EA_PRS_R2.jpg" 
            alt="R2 for EA PRS" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://www.nature.com/articles/nrg.2017.104'&gt;
            Plomin &amp; von Stumm (2018)
        &lt;/a&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='display:none;margin-left:12%;margin-right:12%;'&gt;
        &lt;img src="pics/EA_PRS_perc.png" 
            alt="Distribution of educational success in the quintiles by EA PRS (from Lee et al., 2018)" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
            href='https://www.nature.com/articles/s41588-018-0147-3'&gt;
            Lee et al. (2018)    
        &lt;/a&gt;
    &lt;/div&gt;
&lt;/div&gt;
]

---
class: two-columns

# C+T

.left-col.pad-right-020[
Including all SNPs in the polygenic score
would bring too much noice. How do we account for that?
The initial approach was to use only independent 
and most relevant SNPs: clumping and thresholding (C+T), 
also known as prunning and thresholding (P+T)

.indent[
Clumping = selecting the most predictive SNP
within a window of the genome

Thresholding = selecting the SNPs with p-values
below specified threshold. Different thresholds produce 
different polygenic scores
]

C+T approach is implemented in PLINK, PRSice-2, PRSoS
]

&lt;!-- How to choose p-threshold?&lt;br&gt;
- select a threshold that produces the most predictive PRS;
- or report multiple p-thresholds --&gt;

.right-col.center[
&lt;div class="my-carousel" style='margin-left:7%;margin-right:7%;'&gt;
        &lt;img src="pics/PRSice_barplot.png" 
            alt="Thresholding barplot" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://www.prsice.info/step_by_step/'&gt;
            PRSice
        &lt;/a&gt;
&lt;/div&gt;

]

---
class:two-columns

# Bayesian approach

.left-col.pad-right-020[
The newer approach is to adjust the SNP effect sizes by
highlighting the likely causal SNPs and suppressing the rest

Bayesian methods are used to infer the causal SNPs from the GWAS summary statistics combined with the data about the distance between the SNPs in genome

The SNP effect sizes are adjusted by the degree of their
importance and then used to compute a single polygenic score

Various forms of this approach are implemented in LDpred, SBayesR, PRS-CS, lassosum

]

.right-col.center[
&lt;div class="my-carousel"&gt;
        &lt;img src="pics/PRScs.jpg" 
            alt="Performance of PRS-CS compared to other methods" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://www.nature.com/articles/s41467-019-09718-5'&gt;
            Ge et al. (2019) PRS-CS
        &lt;/a&gt;
&lt;/div&gt;

]

---
class: two-columns
# GWAS statistics for a polygenic score

.left-col.pad-right-020[
It is necessary to obtain the summary
statistics from the GWAS to compute 
a polygenic score. The access procedure is
usually explained in the GWAS paper

.indent[
Some repositories:

[PGS Catalog](https://www.pgscatalog.org/): mainly biological and physiological traits &amp;mdash; free access

[Psychiatric Genomics Consortium](https://www.med.unc.edu/pgc/download-results/): ADHD, ASD, MDD, SCZ, SUD &amp;mdash; free access

[Social Science Genetic Association Consortium](https://www.thessgac.org/pgi-repository): cognitive traits, health-related behaviors, personality, well-being, fertility, sexual development &amp;mdash; access through an account linked to the institutional e-mail

]

]

.right-col.center[
&lt;div class="my-carousel center"&gt;
    &lt;div class="my-carousel-item"&gt;
        &lt;img src="pics/extGWAS-data-access.png" 
            alt="The data access statement from the ExtGWAS paper (Karlsson-Linner et al., 2021)" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://www.nature.com/articles/s41593-021-00908-3#data-availability'&gt;
            Karlsson-Linner et al. (2021)
        &lt;/a&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='display:none;'&gt;
        &lt;img src="pics/PGS-catalogue.png" 
            alt="The title page of PGS Catalog" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
            href='https://www.pgscatalog.org/'&gt;
            PGS Catalog    
        &lt;/a&gt;
    &lt;/div&gt;
    &lt;!-- distributions of the phenotype in PRS quantiles --&gt;
&lt;/div&gt;
]

---
class:two-columns
# One research application

.left-col.pad-right-020[
Polygenic scores are useful in the study of gene-environmental interplay. Genetic nurture effect is the effect of a parental genotype on the child's trait mediated by the environment

When a family trio is genotyped, it is possible to recover which parental alleles were passed to the child and which were not. The association between the child's trait and the polygenic score for non-transmitted alleles is considered as evidence of genetic nurture

Significant genetic nurture effect was discovered for educational outcomes &lt;a href='https://www.sciencedirect.com/science/article/abs/pii/S0002929721002780'&gt;
(Wang et al., 2021, meta-analysis)
&lt;/a&gt;


&lt;!-- &lt;a class='source-link' href='https://www.science.org/doi/abs/10.1126/science.aan6877'&gt;
Kong et al. (2018),&lt;/a&gt; 
&lt;a class='source-link' href='https://www.cambridge.org/core/journals/twin-research-and-human-genetics/article/nature-of-nurture-using-a-virtualparent-design-to-test-parenting-effects-on-childrens-educational-attainment-in-genotyped-families/EA86B33A8224E231EE99B802691634FC'&gt;
Bates et al. (2018),&lt;/a&gt; 
&lt;a class='source-link' href='https://www.cambridge.org/core/journals/twin-research-and-human-genetics/article/social-competence-in-parents-increases-childrens-educational-attainment-replicable-geneticallymediated-effects-of-parenting-revealed-by-nontransmitted-dna/A3686F6B876F33BA210EEB178864A8C8'&gt;
Bates et al. (2019),&lt;/a&gt; --&gt;
]

.right-col.center[
&lt;div class="my-carousel"&gt;
        &lt;img src="pics/gen_nurture_hart.jpg" 
            alt="Illustration of genetic nurturing model" 
            width='100%'
            class='pink'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://www.nature.com/articles/s41539-020-00079-z'&gt;
            Hart et al. (2021)
        &lt;/a&gt;
&lt;/div&gt;

]

---
class:two-columns

# Implications and controversy

.left-col.pad-right-020[
Polygenic scores are becoming increasingly accessible. Their implications for the clinical practice, public health and social policy are being studied and discussed

The controversial practices that involve polygenic scores include: selecting an embryo with the 'best' genetic setup and comparing genetic setup of different ethnic groups

]

.right-col.center[
&lt;div class="my-carousel center"&gt;
    &lt;div class="my-carousel-item" style='padding-left:10%;padding-right:10%;'&gt;
        &lt;img src="pics/implications.png" 
            alt="Turley et al. (2021) Problems with Using Polygenic Scores to Select Embryos" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
           href='https://www.nejm.org/doi/full/10.1056/NEJMsr2105065'&gt;
            Turley et al. (2021)
        &lt;/a&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='display:none;'&gt;
        &lt;img src="pics/controversy.png" 
            style = "background-color:#3e3d3d;"
            alt="Freese et al. (2019, preprint) Problems with a Causal Interpretation of Polygenic Score Differences between Jewish and non-Jewish Respondents in the Wisconsin Longitudinal Study" 
            width='100%'
        /&gt;
        &lt;br&gt;
        &lt;a class='source-link'
            href='https://osf.io/preprints/socarxiv/eh9tq/'&gt;
            Freese et al. (2019, preprint)
        &lt;/a&gt;
    &lt;/div&gt;
    &lt;div class="my-carousel-item" style='display:none;'&gt;
    &lt;div style='display:grid;grid:1fr 1fr/1fr 1fr;gap:10px 10px;justify-items:center;align-items:center;'&gt;
        &lt;img src="pics/genetic-lottery.jpeg" 
            style = "grid-row:1/3;grid-column:1/2"
            alt="Harden (2021) Genetic lottery" 
            width='100%'
        /&gt;
        &lt;img src="pics/genetic-lottery-review-1.png" 
            style = "grid-row:1/2;grid-column:2/3"
            alt="" 
            width='100%'
        /&gt;
        &lt;img src="pics/genetic-lottery-review-2.png" 
            style = "grid-row:2/3;grid-column:2/3"
            alt="" 
            width='100%'
        /&gt;
    &lt;/div&gt;
    &lt;br&gt;
    &lt;a class='source-link'
        href='https://press.princeton.edu/books/hardcover/9780691190808/the-genetic-lottery'&gt;
        Harden (2021)&lt;/a&gt;,
    &lt;a class='source-link'
        href='https://www.americanscientist.org/article/the-house-always-wins'&gt;
        Comfort (2022)&lt;/a&gt;,
    &lt;a class='source-link'
        href='https://massivesci.com/articles/genetic-lottery-review-paige-harden-kevin-bird/'&gt;
        Bird (2021)&lt;/a&gt;
    &lt;/div&gt;
&lt;/div&gt;
]

---
class:two-columns

# Summary on polygenic scores

.left-col.pad-right-020.height-400px.align-to-the-top[
&lt;h3 class='white'&gt;Advantages&lt;/h3&gt;

.indent[
Polygenic score is as powerful as its GWAS&lt;br&gt;
Small effects can be detectable in a sample of hundreds

It is very easy to use once computed&lt;br&gt;
It is just like any another variable in the data table

Multiple applications&lt;br&gt;
The score can be included in standard models, e.g. regression or SEM
]
]

.right-col.pad-left-020.height-400px.align-to-the-top[
&lt;h3 class='white'&gt;Disadvantages&lt;/h3&gt;

.indent[
Only as good as corresponding GWAS&lt;br&gt;
GWAS summary statistics are noisy estimates, especially for rare SNPs

Predictive power is limited when the ancestry of the target population 
doesn't match the GWAS population, and many GWAS samples are predominantly white

Causal SNPs are mixed with non-causal&lt;br&gt;
Non-causal SNP that is close to a causal SNP in the genome adds 
to the noise in the polygenic score

High potential for misuse
]
]

---
name: glossary
class:two-columns glossary

# Glossary and abbreviations

.left-col.pad-right-020[
.indent[

Additive model of genetic effect&lt;br&gt;
the model that assumes that each copy of an allele independently adds to the trait

Allele&lt;br&gt;
an alternative form of a specific genetic variant

Allele frequency&lt;br&gt;
the relative frequency of an allele of 
a specific genetic variant in a population

Base pair&lt;br&gt;
a pair of complementary linked nucleotides

]
]

.right-col.pad-left-020[
.indent[

Biological annotation of associations&lt;br&gt;
the inference of biological mechanisms of SNP effects
from the information about the function of genomic 
region where they are located

Causal SNP&lt;br&gt;
a SNP that is assumed to have a biological effect on a phenotype

Clumping + thresholding (C+T),
or prunning + thresholding (P+T)&lt;br&gt;
the process of selection of the independent
and most relevant SNPs to use for the computation
of a polygenic score

]
]

---
class:two-columns glossary

# Glossary and abbreviations

.left-col.pad-right-020[
.indent[

DNA&lt;br&gt;
deoxyribonucleic acid, a molcule that carries genetic code

EA, or educational attainment&lt;br&gt;
number of years spent by a person on formal education

Effect allele&lt;br&gt;
an allele whose allelic score is 1

Externalizing behaviors&lt;br&gt;
a variety of disruptive behaviors related to the lack of self-regulation, 
incl. aggression, substance use and risk taking


]
]

.right-col.pad-left-020[
.indent[

Gene&lt;br&gt;
a region in genome that encodes a biologically active molecule (protein or RNA)

Gene expression&lt;br&gt;
a process of building a protein or RNA product from the blueprint of a gene

Genetic correlation&lt;br&gt;
the overlap between genetic variance of two traits

Genetic nurture&lt;br&gt;
an effect of parental genotype on a child's trait mediated by environment

]
]

---
class:two-columns glossary

# Glossary and abbreviations

.left-col.pad-right-020[
.indent[

Genetic variation&lt;br&gt;
variability of genetic code between the organisms of the same species

Genome&lt;br&gt;
complete genetic information of an organism

Genomic SEM&lt;br&gt;
a method that estimates the associations between
SNPs and a latent variable

Genotype&lt;br&gt;
the combination of alleles carried by an individual at a specific locus
or in multiple loci across the genome

]
]

.right-col.pad-left-020[
.indent[
Genotype imputation&lt;br&gt;
the inference of alleles in a locus that was unmeasured from the alleles in the measured loci of the same person

Genotyping&lt;br&gt;
the measurement of the genotype from a DNA sample

Genotype score&lt;br&gt;
the sum of allele scores in a locus

GWAS, or genome-wise association study&lt;br&gt;
the study of associations between a phenotype and multiple genetic 
variants across the genome

]
]

---
class:two-columns glossary

# Glossary and abbreviations

.left-col.pad-right-020[
.indent[

GWAS-by-subtraction&lt;br&gt;
a form of Genomic SEM that estimates the association between SNPs and a trait while removing genetic variance linked to another trait

GWAS hit&lt;br&gt;
a specific SNP effect that has reached the 
level of statistical significance in a GWAS

GWAS summary statistics&lt;br&gt;
the regression results from a GWAS,
stored in a table that lists variants' ID, location,
effect size and statistical significance

]
]

.right-col.pad-left-020[
.indent[

HG&lt;br&gt;
human genome


Indel&lt;br&gt;
incertion or deletion of a base pair

LD, or linkage disequilibrium&lt;br&gt;
non-random association between two genetic variants
that are close in the genome

LD reference panel&lt;br&gt;
a genotype panel that is used to estimate how often two
genetic variants segregate in the process of Mendelian ingeritance

]
]

---
class:two-columns glossary

# Glossary and abbreviations

.left-col.pad-right-020[
.indent[

Locus (pl.: loci)&lt;br&gt;
a specific location in the genome

Meta-GWAS, or GWAS meta-analysiss&lt;br&gt;
a meta-analysis of the effects from multiple GWASes

Microarray DNA chip&lt;br&gt;
a chip with an array of microscopic DNA spots that are sensitive
to the alleles of specific genetic variants

Nucleotide&lt;br&gt;
a building block of DNA, organic molecule consisting of a sugar, 
a phosphate group and a nitrogenous base

]
]

.right-col.pad-left-020[
.indent[
OR, or odds ratio&lt;br&gt;
the ratio between the odds of having the disease and the odds of not having 
the disease conditional on having a specific allele

Phenotype&lt;br&gt;
a measureable trait

Polygenic score, or polygenic index&lt;br&gt;
an index of genetic risk or propensity for a trait
calculated as a sum of individual's alleles weighed
by the size of their effects from a GWAS


Polymorphism, or genetic variant&lt;br&gt;
genomic region where alternative forms exist in a population


]
]

---
class:two-columns glossary

# Glossary and abbreviations

.left-col.pad-right-020[
.indent[



QC, or genotype quality control&lt;br&gt;
procedures performed on the genotype data to ensure its reliability

Reference genome&lt;br&gt;
a digital database that represents an idealized genetic sequence
of a single person to serve as an index of genetic features

RNA&lt;br&gt;
ribonucleic acid, a one-stranded sequence of nucleotides

Structural variation&lt;br&gt;
variation of the chromosomal structure

]
]

.right-col.pad-left-020[
.indent[

SEM, or structural equation modelling&lt;br&gt;
a multivariate statistical technique that
estimates the associations between measured and
unmeasured (latent) variables from the covariance matrix


SNP, or single nucleotide polymorphism&lt;br&gt;
a substitution of one base pair
in a specific genomic location

SNP heritability&lt;br&gt;
the amount of the trait's variance explained by all SNPs in a GWAS




]
]

---
class:last-slide

&lt;div class='panel'&gt;
&lt;a href='mailto:ivan.a.voronin@gmail.com'&gt;&lt;i class="fa-regular fa-envelope"&gt;&lt;/i&gt; ivan.a.voronin@gmail.com&lt;br&gt;&lt;/a&gt;
&lt;a href='https://twitter.com/IvanVoroninPhD'&gt;&lt;i class="fa-brands fa-twitter"&gt;&lt;/i&gt; IvanVoroninPhD&lt;br&gt;&lt;/a&gt;
&lt;a href='https://github.com/IvanVoronin'&gt;&lt;i class="fa-brands fa-github"&gt;&lt;/i&gt; IvanVoronin&lt;/a&gt;
&lt;br&gt;
&lt;br&gt;
With gratitude to Pavel Voronin for his insightful advice and thorough edits
&lt;/div&gt;
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
